draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -148,6 +157,7 @@ $row-height: 60px;
   padding: 0 $ca-grid * 0.8;
   display: flex;
   align-items: center;
+  box-shadow: none;
```

Comment:
does this need to be added here? Cells already didn't have a box shadow, so it doesn't seem necessary to add this. Or is this to override where it's added somewhere else using a CSS selector? I'm lacking context on this change.

Reply:
oh geez not at all

Reply:
Sorry! Was a relic of playing around to see if it was the cell wreaking havoc!